src/main/java/org/mockito/internal/configuration/plugins/PluginLoader.java

```diff
@@ -51,20 +59,22 @@ public PluginLoader(PluginSwitch pluginSwitch) {
         try {
             resources = loader.getResources("mockito-extensions/" + service.getName());
         } catch (IOException e) {
-            throw new MockitoException("Failed to load " + service, e);
+            // use RuntimeException instead MockitoException, because MockitoException depends on plugins and can
+            // break stack track
+            throw new RuntimeException("Failed to load " + service, e);
         }
 
         try {
             String foundPluginClass = new PluginFinder(pluginSwitch).findPluginClass(Iterables.toIterable(resources));
             if (foundPluginClass != null) {
-                Class<?> pluginClass = loader.loadClass(foundPluginClass);
+                Class<?> pluginClass = Class.forName(foundPluginClass);
```

Comment:
I wonder how  old statement could work in this case: 

1. The `MockUtil` is created first time from class which is loaded by Application class loader.  Important that no other Mockito classes haven been used before creating `MockUtil`. So it means that the `MockUtil` class will be loaded by this class loader. 
2. During class initialisation the `Plugins.getMockMaker` is  called, an `Plugins` class also loaded by application class loader.
3. Next step `PluginRegistry` - and again application class loader is used. 
4. And now the `PluginLoader` is called, which uses completed different context class loader to load plugin. Okay, plugin can be loaded and may even cast and return, but how it could be assign to field? 

By application class loader I mean that it's class loader which is used to load first class with `main()`method. A  context class loader - is a new class loader which is create via `new` and doesn't have application class loader as a parent